### PR TITLE
[process][m]: refactored to python script and updated all related data

### PR DESCRIPTION
**Pull request by Mikanebu**
* refactored to python script
* updated readme
* added dataflow and generated new datapackage.json REMOVED
* updated data itself, 3k new cities added
* added github actions with period yearly basis
* added requirements.txt

### CODE REVIEW

File: datapackage.json [hunk 1]
```diff
@@ -1,87 +1,57 @@
...
-  "name": "world-cities",
-  "title": "Major cities of the world",
+  "bytes": 1019186,
+  "count_of_rows": 26467,
   "description": "List of the world's major cities (above 15,000 inhabitants)",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/datasets/world-cities.git"
-  },
-  "sources": [
```
Comment (anuveyatsu) [inline datapackage.json]: @Mikanebu pls bring it back if we're still using Geonames as a source.

File: datapackage.json [hunk 2]
```diff
@@ -1,87 +1,57 @@
...
-      "name": "Geonames",
-      "path": "http://www.geonames.org/",
-      "title": "Geonames"
-    }
-  ],
-  "keywords": [
-    "geodata",
-    "city"
-  ],
-  "licensces": [
```
Comment (anuveyatsu) [inline datapackage.json]: Please keep the license and i can see that there was a typo previously.

File: datapackage.json [hunk 3]
```diff
@@ -1,87 +1,57 @@
...
       "path": "data/world-cities.csv",
-      "format": "csv",
-      "mediatype": "text/csv",
+      "profile": "tabular-data-resource",
       "schema": {
         "fields": [
           {
+            "format": "default",
             "name": "name",
-            "description": "English name of the city",
```
Comment (anuveyatsu) [inline datapackage.json]: All these descriptions are useful metadata, please, don't remove them @Mikanebu

**Review by anuveyatsu — CHANGES_REQUESTED** [review]
@Mikanebu I left comments

File: datapackage.json (hunk 3 above)
Comment (Mikanebu) [reply datapackage.json]: Kept previous version of datapackage.json

File: datapackage.json (hunk 2 above)
Comment (Mikanebu) [reply datapackage.json]: Kept previous version of datapackage.json

File: datapackage.json (hunk 1 above)
Comment (Mikanebu) [reply datapackage.json]: Kept previous version of datapackage.json

**Comment by Mikanebu** [comment]
@anuveyatsu Please, check again and everything is fine, pls merge.

**Pull request merged**